### PR TITLE
Jupyterlab Extension Install Fix

### PR DESCRIPTION
**Pull request by akarukappadath**
Users were unable to install jupyterlab extensions after cluster creation. The reason for this was that a previous PR changed the extensions to be installed as root, when they were originally installed as jupyteruser. This PR reverts this change. 

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookCustomizationSpec.scala [hunk 1]
```diff
@@ -94,6 +94,21 @@ final class NotebookCustomizationSpec extends FreeSpec
...
     }
 
+    "should allow users to install extensions after cluster creation" in {
+      withProject { project => implicit token =>
+        withNewCluster(project) { cluster =>
+          withWebDriver { implicit driver =>
+            withNewNotebook(cluster) { notebookPage =>
+              val query = """!jupyter labextension install @jupyterlab/toc"""
+              val result = notebookPage.executeCell(query).get
+              println(result)
```
Comment (vkumra-broad) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookCustomizationSpec.scala]: remove the `print` statement

Comment (akarukappadath) [reply automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookCustomizationSpec.scala]: oops 😅

**Comment by vkumra-broad** [comment]
LGTM, I'll approve once tests pass.

**Comment by rtitle** [comment]
Hm, I think there was a reason I added `-u root` when installing Lab extensions in https://github.com/DataBiosphere/leonardo/pull/787. I'm not 100% sure though. Before merging this can we do a manual test: spin up a fiab from this branch, create a cluster, open JupyterLab, verify the token refresh is still happening. If that works I am satisfied.

**Comment by akarukappadath** [comment]
jenkins retest

**Comment by akarukappadath** [comment]
jenkins retest

**Comment by akarukappadath** [comment]
jenkins retest

**Comment by akarukappadath** [comment]
jenkins retest

**Review by rtitle — APPROVED** [review]
lgtm

**Pull request merged**